### PR TITLE
Fixes issue #1547

### PR DESCRIPTION
**Pull request by hessio**
Removed hard coded secret key

Signed-off-by: hessio <patrick.hession@seagate.com>

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

### CODE REVIEW

File: doc/integrations/parlAI/application.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 
 ACCESS_KEY = "AKIAtEpiGWUcQIelPRlD1Pi6xQ"
```
Comment (mukul-seagate11) [inline doc/integrations/parlAI/application.py:43]: ACCESS_KEY also need to be parameterized

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: doc/integrations/parlAI/application.py (hunk 1 above)
Comment (hessio) [reply doc/integrations/parlAI/application.py:43]: Ok

File: doc/integrations/parlAI/application.py [hunk 2]
```diff
@@ -41,7 +41,7 @@
...
-SECRET_KEY = "YNV6xS8lXnCTGSy1x2vGkmGnmdJbZSapNXaSaRhK"
+SECRET_KEY = <SECRET_KEY> 
 host = "http://192.168.2.102"
```
Comment (mukul-seagate11) [inline doc/integrations/parlAI/application.py:45]: parameterized IP address as well as it might be a security risk

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

**Review by johnbent — APPROVED** [review]
lgtm

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**